content/source/docs/enterprise/vcs/troubleshooting.html.md

```diff
@@ -85,6 +85,20 @@ The domain name for Terraform Enterprise changed on 02/22 at 9AM from atlas.hash
 
 The fix is to update the OAuth Callback URL in your VCS provider to use app.terraform.io instead of atlas.hashicorp.com.
 
+### Changing the API URL for a VCS provider
+
+On rare occasions, you might need TFE to change the URL it uses to reach your VCS provider. This usually only happens if you move your VCS server or the VCS vendor changes their supported API versions.
+
+TFE does not allow you to change the API URL for an existing VCS connection, but you can create a new VCS connection and update existing resources to use it. This is most efficient if you script the necessary updates using TFE's API. In brief:
+
+1. Configure a new VCS connection with the updated URL.
```

Comment:
I'm thinking maybe we link to the index page for the section here?

Reply:
👍🏼 

```suggestion
1. [Configure a new VCS connection](./index.html) with the updated URL.
```